plugins/kubernetes/src/api/types.ts

```diff
@@ -33,6 +33,7 @@ export interface KubernetesApi {
   getClusters(): Promise<
     {
       name: string;
+      displayName?: string | undefined;
```

Comment:
`displayName?: string` is enough. The `?` already means the same.

Basically:
`displayName?: string` == `displayName: string | undefined`

Reply:
done